ui/src/viewers/PdfViewer.jsx

```diff
@@ -16,13 +16,16 @@ import {queryEntities} from 'src/actions';
 import {selectEntitiesResult} from 'src/selectors';
 
 import './PdfViewer.scss';
+import TextViewer from 'src/viewers/TextViewer';
+
 
 class PdfViewer extends Component {
-  static TextMode(props) {
-    const {document, result} = props;
-    return result.total > 0 && <React.Fragment>
+
+  static TextMode(props){
+    const {document, result, page} = props;
+    return  result.total > 0 && <React.Fragment>
       <PagingButtons document={document} numberOfPages={result.total}/>
-      <Pre>{result.results[0].getProperty('bodyText').toString()}</Pre>
+      <TextViewer document={result.results[page-1]} noStyle={true}/>
```

Comment:
Why do we need the text viewer here?

Reply:
Because TextViewer has a functionality to load the `/content` if it missing